modules/index/src/main/java/org/apache/ignite/internal/index/IndexBuildController.java

```diff
@@ -541,7 +545,12 @@ private static MvPartitionStorage mvPartitionStorage(
             int tableId,
             int partitionId
     ) {
-        MvPartitionStorage mvPartition = mvTableStorage.getMvPartition(partitionId);
+        MvPartitionStorage mvPartition;
```

Comment:
Maybe in a separate method to make it look more elegant?

Reply:
Current method already has just that try/catch and an assertion, this is just a few lines, so there doesn't seem to be any readability problem that needs to be solved. If we introduce a new method, we'll have 2 tiny methods, both of which will be almost of the same size as current one; they will have almost identical meaning, but the outer will additionally make the assertion. I find it difficult to invent meaningful names for them.

To sum up, I'd prefer to leave the current method in its current form.